### DISCUSSION TITLE
Configuration of Dbeaver

### DISCUSSION THREAD

**Discussion by Slyleglobe** [dbeaver/dbeaver#10755] [Q&A]
Hi all,

I had a very old version of Dbeaver installed on my computer and decided to update it with the last one. However, I'm not anymore able to connect me to the Database i was working on.
I now have a message "Got minus one from a read call" with a SQL Error [17002][08006].

I'm a beginner with Dbeaver and completely lost. Would you have an idea to help me? Otherwise, is there a way to come back to the previous version installed?

Thanks a lot for your help,
Sylvain

**Comment by serge-rider** [comment]
L:ooks liek an issue with Oracle database connectivity. What is your Oracle server version?
Try to:
- Use another Oracle driver version. Probably you had some other driver version confgured in old DBeaver installation, see https://github.com/dbeaver/dbeaver/wiki/Database-drivers for driver config instructions.
- You can downgrade DBeaver (see https://dbeaver.io/files). However it doesn't guarantee that the problem will be solved. Most likely it happens beacuse of wrong client driver version.
